### ISSUE TITLE
Code sign OSX release

### ISSUE THREAD

**Issue by micahflee** [micahflee/onionshare#151]
I want to code sign the OSX release. Hopefully I can do it with the same StartSSL cert I'm getting to sign the Windows release (#72), but I've never done this before.

Here are some resources that might be helpful:

https://developer.apple.com/library/mac/Documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html
https://discussions.apple.com/thread/2382180?tstart=0

Is there a bug in PyInstaller that will cause me headaches? http://www.pyinstaller.org/ticket/812

**Comment by micahflee** [comment]
Just confirmed that, while I can codesign OSX apps with a StartSSL certificate, it doesn't satisfy Gatekeeper. It looks like you need to join the Mac Developer Program for $99/year for that to work. https://twitter.com/FredericJacobs/status/571047607155466240

**Comment by micahflee** [comment]
PyInstaller 2.1, the current stable version, doesn't support OSX codesigning. It looks like there's [documentation](https://github.com/pyinstaller/pyinstaller/wiki/Recipe-OSX-Code-Signing) for OSX codesigning, but the `--osx-bundle-identifier` flag isn't in the stable release. And it looks like the codesigning bug has been closed too https://github.com/pyinstaller/pyinstaller/issues/812 but it's slated for version 2.2.

I just tried using pyinstaller from the master branch instead of the 2.1 release, and ran into some issues. Not to mention it's not stable yet.

Maybe I should switch OSX packaging to py2app.
